src/vue/vuex-modules/data/auth-module.js

```diff
@@ -12,8 +12,12 @@ export default class AuthModule extends Module {
     this._externalLoginUrl = null
     this._externalLogoutUrl = null
     if (this._auth) {
-      this._externalLoginUrl = this._auth.loginUrl()
-      this._externalLogoutUrl = this._auth.logoutUrl()
+      try {
```

Comment:
this is just to keep older code auth module implementations from failing until all the code gets in sync. See #388